src/modules/Thread/components/ThreadList/ThreadListItemContent.tsx

```diff
@@ -91,6 +93,7 @@ export default function ThreadListItemContent({
   );
   const supposedHoverClassName = supposedHover ? 'sendbird-mouse-hover' : '';
   const isReactionEnabledInChannel = isReactionEnabled && !channel?.isEphemeral;
+  const isOgMessageEnabledInGroupChannel = channel.isGroupChannel() && config.groupChannel.enableOgtag;
```

Comment:
Considering the type of `channel` ([declared at the above](https://github.com/sendbird/sendbird-uikit-react/pull/603/files#diff-8645091ee9cc99dd8dd55fd33017b226611fc495af7fe8d0178757fe00c075f0R36) in this file) is `GroupChannel`, I think checking these two conditions should be enough.